dev/demo_web.ex

```diff
@@ -26,6 +26,10 @@ defmodule DemoWeb do
       use Phoenix.LiveView,
         layout: {DemoWeb.LayoutView, "live.html"}
 
+      if Version.match?(System.version(), "<= 1.11.0") do
+        :ok = :application.load(:phoenix_live_view)
```

Comment:
I'm not happy with this line but I'm getting the following error without it:

```
= Compilation error in file dev/demo_web/live/app_live/index.ex ==
** (Version.InvalidVersionError) invalid version: ""
    (elixir) lib/version.ex:339: Version.to_matchable/2
    (elixir) lib/version.ex:181: Version.match?/3
    dev/demo_web/live/app_live/index.ex:2: (module)
    (stdlib) erl_eval.erl:680: :erl_eval.do_apply/6
    (elixir) lib/kernel/parallel_compiler.ex:206: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/6
```

That's because `Application.spec(:phoenix_live_view, :vsn)` returns `nil`

Reply:
What if instead we remove the version check below? In hindsight we don't need to guard against such things in the example "app". :)

Reply:
Done